ai-platform-unified/notebooks/unofficial/hyperparameter_tuning/distributed-hyperparameter-tuning.ipynb

```diff
@@ -0,0 +1,879 @@
+{
```

Comment:
<p>In Overview, refer to this as the "mirrored strategy" for distributed training.</p><p> </p><p>There are two Vertex hyperparameter tuning services. Be specific which one is used here.</p><p> </p><p>Inconsistent capitalization:</p><p>Vertex AI Hyperparameter</p><p>Vertex AI hyperparameter</p><p> </p><p>Except when referring to Vertex AI as a platform, the AI part is dropped.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Done